docs/docs/chitchat-faqs.mdx

```diff
@@ -2,336 +2,72 @@
 id: chitchat-faqs
 sidebar_label: Chitchat and FAQs
 title: Chitchat and FAQs
+abstract: FAQ assistants are the simplest assistants to build and typically the first kind of assistant anyone builds. This page is a guide to the concepts and training data you need to handle non-contextual questions like FAQs and chitchat.
 ---
 
-import useBaseUrl from '@docusaurus/useBaseUrl';
+[FAQs](./glossary.mdx#faqs) and [chitchat](./glossary.mdx#chitchat) are two cases where the conversational assistant's answer is a single message,
+and the assistant should always answer the same way,
+no matter what has happened previously in the conversation.
+For example, in the
+following conversation, every question's answer is independent of context and
+could
+just as easily be answered on its own:
 
-FAQ assistants are the simplest assistants to build and a good place to get started.
-These assistants allow the user to ask a simple question and get a response. You're
-going to build a basic FAQ assistant using features of Rasa designed specifically
-for this type of assistant. You can handle chitchat in the same way.
-
-In this section you will read about the following topics:
-
-* [Responding to simple messages](./chitchat-faqs.mdx#responding-to-simple-messages)
-  with the `MemoizationPolicy`
-
-* [Handling FAQs](./chitchat-faqs.mdx#handling-faqs-using-a-response-selector)
-  using a `ResponseSelector`
-
-## Prerequisites
-
-You should [first install Rasa](installation.mdx) and
-[prototype an assistant](prototype-an-assistant.mdx) to make sure
-you understand the basic concepts of intents, stories and domains.
-
-For this tutorial, you can create a new Rasa project using the CLI:
-
-```bash
-rasa init
+```yaml
```

Comment:
there will be a chat component available soon which lu is working on, we should use that when it's ready